src/parser/mod.rs

```diff
@@ -1049,18 +1049,18 @@ impl<'a> Parser<'a> {
             | Keyword::CURRENT_USER
             | Keyword::SESSION_USER
             | Keyword::USER
-                if dialect_of!(self is PostgreSqlDialect | GenericDialect) =>
```

Comment:
The following parts are auto formatted by the command: `cargo format`.

Reply:
I wonder why it just happened now 🤔  Is it possible you are using nightly rust 🤔

Reply:
@alamb aha, I'm using 1.82.0 on my side.